src/Workspaces/Core/Portable/FindSymbols/FindReferences/Finders/PropertySymbolReferenceFinder.cs

```diff
@@ -183,6 +187,53 @@ private static bool IsForEachProperty(IPropertySymbol symbol)
             return locations.ToImmutableAndFree();
         }
 
+        private async Task<ImmutableArray<FinderLocation>> FindConditionalAccessReferencesAsync(
+            IPropertySymbol symbol, Document document, SemanticModel semanticModel,
+            FindReferencesSearchOptions options, CancellationToken cancellationToken)
+        {
+            if (options.AssociatePropertyReferencesWithSpecificAccessor)
+            {
+                // Looking for individual get/set references.  Don't find anything here. 
+                // these results will be provided by the PropertyAccessorSymbolReferenceFinder
+                return ImmutableArray<FinderLocation>.Empty;
```

Comment:
is that true?  can you add a test that demonstrates this?  its should probably go in FindReferencesTests.AccessorSymbols.vb

Reply:
This comment was copy-pasted from already existing source code in this class. The duplication of the comment is cleaned up. I will take a look if I can find out more about the comment to demonstrate it/ make it easier to verify.